src/main/java/com/mraof/minestuck/data/loot_table/MSChestLootTables.java

```diff
@@ -64,7 +69,7 @@ public void accept(BiConsumer<ResourceLocation, LootTable.Builder> lootProcessor
 						.add(LootItem.lootTableItem(Items.DIAMOND).setWeight(1).setQuality(3).apply(SetItemCountFunction.setCount(UniformGenerator.between(1, 3))))
 						.add(LootItem.lootTableItem(Items.PUMPKIN).setWeight(3).setQuality(-1).apply(SetItemCountFunction.setCount(UniformGenerator.between(1, 5))))
 						.add(LootItem.lootTableItem(Items.WRITABLE_BOOK).setWeight(2).setQuality(1).apply(SetItemCountFunction.setCount(UniformGenerator.between(0, 1))))
-						.add(LootItem.lootTableItem(MSItems.BLANK_DISK.get()).setWeight(5).setQuality(3).apply(SetItemCountFunction.setCount(UniformGenerator.between(1, 2))))
+						.add(LootItem.lootTableItem(MSItems.BLANK_DISK.get()).setWeight(5).setQuality(3).apply(SetItemCountFunction.setCount(UniformGenerator.between(0, 2))))
```

Comment:
Potentially setting the item count to 0 seems a bit odd. Would you consider tweaking the loot entry weight instead?